src/components/pages/Categories/Categories.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="category">
+    <app-category-Post
+      :access="access"
+      :error-message="errorMessage"
+    />
+    <app-category-list
+      :theads="theads"
+      :categories="categories"
+      :access="access"
+    />
```

Comment:
[imo]

pages 側でスタイルを上書きしたい場合こちらに class を付与してスタイルを適用した方が良いと思います。
このファイルの <style> タグに指定されているスタイルが、どこの要素に当たっているのかが分かりづらいためです。

```suggestion
    <app-category-Post
      :access="access"
      :error-message="errorMessage"
      class="category-form"
    />
    <app-category-list
      :theads="theads"
      :categories="categories"
      :access="access"
      class="category-list"
    />
```